data/src/main/scala/io/prediction/data/storage/PEvents.scala

```diff
@@ -179,4 +179,8 @@ trait PEvents extends Serializable {
     */
   @DeveloperApi
   def write(events: RDD[Event], appId: Int, channelId: Option[Int])(sc: SparkContext): Unit
+
+  @DeveloperApi 
+  def delete(eventIds: RDD[String], appId: Int, channelId: Option[Int])(sc: SparkContext): Unit
+
 }
```

Comment:
ready for review

Fixes a problem in the example Cleanup App, where it has to do `rdd.collect` to actually delete events. Collect does not scale.